### PR TITLE
Make sure "default" is selected at start-up

### PR DESCRIPTION
**Pull request by dafeda**
Resolves: #4964 

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Updated documentation
- [ ] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Comment by oyvindeide** [comment]
> Current case does not show New case_0, New case_1 etc.:

This _might_ be because the signals are not propagated correctly, did you try restarting? Not that is the actual solution, but points to the problem.

**Comment by oyvindeide** [comment]
> This is how it now looks at start-up, i.e., Current case shows New case:

Is that what we want though? Dont we want the option of creating a new case when starting up, but naming it at the same time? Because now we get a case called `New case` if I read it correctly?

**Comment by dafeda** [comment]
> > This is how it now looks at start-up, i.e., Current case shows New case:
> 
> Is that what we want though? Dont we want the option of creating a new case when starting up, but naming it at the same time? Because now we get a case called `New case` if I read it correctly?

In this minimal user-story (😬 ), do we want users to have to do step 2?

1. ert gui poly.ert
2. Create new case?
3. Start simulation

**Comment by dafeda** [comment]
> > Current case does not show New case_0, New case_1 etc.:
> 
> This _might_ be because the signals are not propagated correctly, did you try restarting? Not that is the actual solution, but points to the problem.

Problem is still there after restart.

**Comment by oyvindeide** [comment]
> > > This is how it now looks at start-up, i.e., Current case shows New case:
> > 
> > 
> > Is that what we want though? Dont we want the option of creating a new case when starting up, but naming it at the same time? Because now we get a case called `New case` if I read it correctly?
> 
> In this minimal user-story (😬 ), do we want users to have to do step 2?
> 
>     1. ert gui poly.ert
> 
>     2. Create new case?
> 
>     3. Start simulation

Good point, I think we can stay with the `current case` workflow, but why change `default` for `New case` in that case? I would also like the option of typing a new name into that box, which will create a fresh case. And perhaps a warning if running in an existing case.

**Comment by dafeda** [comment]
> > > > This is how it now looks at start-up, i.e., Current case shows New case:
> > > 
> > > 
> > > Is that what we want though? Dont we want the option of creating a new case when starting up, but naming it at the same time? Because now we get a case called `New case` if I read it correctly?
> > 
> > 
> > In this minimal user-story (😬 ), do we want users to have to do step 2?
> > ```
> > 1. ert gui poly.ert
> > 
> > 2. Create new case?
> > 
> > 3. Start simulation
> > ```
> 
> Good point, I think we can stay with the `current case` workflow, but why change `default` for `New case` in that case? I would also like the option of typing a new name into that box, which will create a fresh case. And perhaps a warning if running in an existing case.

I guess we have agreed not to change how this works for now, and so I have renamed "New case" to "default" in this PR.
This PR does not solve everything, but at least the "Current case" drop-down is not empty at start-up.
I would like to merge this as is and then we can continue working on the other GUI-related issues in other PR's.

File: src/ert/gui/simulation/ensemble_smoother_panel.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def __init__(self, ert: EnKFMain, notifier: ErtNotifier):
         facade = LibresFacade(ert)
         layout = QFormLayout()
 
-        self._case_selector = CaseSelector(facade, notifier)
+        self._case_selector = CaseSelector(facade, notifier, show_create_new_case=True)
```
Comment (oyvindeide) [inline src/ert/gui/simulation/ensemble_smoother_panel.py:36]: Shouldnt the `show_create_new_case` property be removed? Not sure it is related to this PR, but if I now run two experiments, I get two ensembles, both named `default`. If I combine this PR with: https://github.com/equinor/ert/pull/4982 i only get one ensemble as I would expect, so perhaps the property is redundant?

Comment (dafeda) [reply src/ert/gui/simulation/ensemble_smoother_panel.py]: I have removed it.

**Review by oyvindeide — APPROVED** [review]
Now works as expected, LGTM!

File: src/ert/gui/ertwidgets/caseselector.py [hunk 2]
```diff
@@ -67,24 +65,21 @@ def populate(self):
...
-            new_case_font.setItalic(True)
-            self.addItem("New case", userData=None)
-            self.setItemData(0, new_case_font, Qt.FontRole)
+        current_index = 0
+        new_case_font = QFont(self.font())
+        new_case_font.setItalic(True)
+        self.addItem("default", userData=None)
+        self.setItemData(0, new_case_font, Qt.FontRole)
+        # The first entry is "default"
+        current_index += 1
```
Comment (dafeda) [inline src/ert/gui/ertwidgets/caseselector.py:74]: Not sure what to do here

**Pull request closed without merging**